django_ibmi/base.py

```diff
@@ -311,15 +226,21 @@ def init_connection_state(self):
         pass
 
     def is_usable(self):
-        # TODO implement is_usable method correctly
+        try:
+            # If connection is closed and unusable, a Programming error will result
+            self.connection.cursor()
+        except pyodbc.ProgrammingError:
+            return False
         return True
 
     def _set_autocommit(self, autocommit):
-        self.connection.autocommit = autocommit
+        with self.wrap_database_errors:
```

Comment:
Just noticed this. We could have probably used that [here](https://github.com/IBM/django-ibmi/blob/master/django_ibmi/base.py#L419-L423) when doing the Python 2 -> Python 3 conversion.